### PR TITLE
Action example

### PR DESCRIPTION
**Pull request by yuokamoto**
Add fibonacci action example in C++ and BP
todo
- add to documentation
- apply changes to UE_tools to generate action

### CODE REVIEW

File: Source/rclUE/Public/rclcUtilities.h [hunk 1]
```diff
@@ -256,4 +257,26 @@ class UROS2Utils : public UBlueprintFunctionLibrary
...
+
+    static builtin_interfaces__msg__Time FloatToROSStamp(const float InTimeSec)
+    {
+        builtin_interfaces__msg__Time stamp;
+        stamp.sec = static_cast<int32>(InTimeSec);
+        stamp.nanosec = (uint64)(InTimeSec * 1e+09f);
+        return stamp;
+    }
+
+    static float ROSStampToFloat(const builtin_interfaces__msg__Time InTimeStamp)
```
Comment (Tadinu) [inline Source/rclUE/Public/rclcUtilities.h]:
```suggestion
    static float ROSStampToFloat(const builtin_interfaces__msg__Time& InTimeStamp)
```

**Review by Tadinu — APPROVED** [review]
Use `int8` for all `for (int i = 0; i < 16; i++)`?

File: Source/rclUE/Private/ROS2ActionClient.cpp [hunk 2]
```diff
@@ -93,6 +93,8 @@ void UROS2ActionClient::UpdateAndSendGoal()
     {
         UE_LOG(LogROS2Action, Error, TEXT("Action Server Unavailable (%s)"), *__LOG_INFO__);
     }
+
+    return ActionServerIsAvailable;
```
Comment (Tadinu) [inline Source/rclUE/Private/ROS2ActionClient.cpp]:
```suggestion
    return bIsActionServerAvailable;
```

**Review by tetorea — APPROVED** [review]

**Comment by yuokamoto** [comment]
> Use `int8` for all `for (int i = 0; i < 16; i++)`?

it seems action uses this msg as goal_id
https://github.com/ros2/unique_identifier_msgs/blob/rolling/msg/UUID.msg

and since bp can't use int8, I added some utils.

**Pull request merged**